docker-compose.yml

```diff
@@ -148,11 +151,64 @@ services:
     command: ["portal"]
     restart: on-failure:3
     healthcheck:
-      test: ["CMD", "curl", "-k", "http://localhost:8444/healthz"]
+      test: ["CMD", "curl", "-k", "-sS", "https://localhost:8444/healthz/ready"]
```

Comment:
same as above - can this fail?

Reply:
These tend to fail when the services fail to come up because of some sort of configuration issue.

Reply:
Updated healthchecks for vpn and portal services to validate HTTP 200 responses, ensuring more reliable detection of service readiness and improved debugging for failure scenarios.